crates/api_structs/src/lib.rs

```diff
@@ -81,31 +82,32 @@ fn do_send_local_notifs(
   // Send the local mentions
   for mention in mentions
     .iter()
-    .filter(|m| m.is_local() && m.name.ne(&user.name))
+    .filter(|m| m.is_local() && m.name.ne(&person.name))
     .collect::<Vec<&MentionData>>()
   {
-    if let Ok(mention_user) = User_::read_from_name(&conn, &mention.name) {
+    // TODO do a local user fetch
+    if let Ok(mention_user_view) = LocalUserView::read_from_name(&conn, &mention.name) {
       // TODO
       // At some point, make it so you can't tag the parent creator either
       // This can cause two notifications, one for reply and the other for mention
-      recipient_ids.push(mention_user.id);
+      recipient_ids.push(mention_user_view.local_user.id);
```

Comment:
Really unfortunate, because in most places in the code recipient_id, creator_id, etc means `person_id`, but with websockets, it means `local_user_id`